solidity/random-beacon/contracts/libraries/DKG.sol

```diff
@@ -445,5 +463,17 @@ library DKG {
         delete self.submittedResultHash;
         delete self.resultSubmitter;
         delete self.submittedResultBlock;
+        self.sortitionPool.unlock();
+    }
+
+    /// @notice Cancels the warm up of group creation process by unlocking
+    ///         the sortition pool.
+    function coolDown(Data storage self) internal {
```

Comment:
Maybe `notifySeedTimedOut` and put this function next to `notifyTimeout`?

Reply:
See https://github.com/keep-network/keep-core/pull/2691/commits/a29a5ad1f24125d53b37cef20129d6c5977764be